### PR TITLE
#1416 Show meaningful error message when email verification not done.

### PR DESCRIPTION
**Pull request by Obtin**
"Fix #1413 : Now if the email is not verified an appropriate message is shown indicating the user the reason why he can't proceed ahead ."
pr

### CODE REVIEW

File: requirements/common.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ drfdocs==0.0.11
 pika==0.10.0
 pickleshare==0.7.4
 Pillow==3.4.2
-psycopg2==2.6.2
+psycopg2==2.7.3.1
```
Comment (RishabhJain2018) [inline requirements/common.txt]: Please remove this change as it is not the part of this issue. Moreover, can you please tell me why you are updating the version ?

Comment (16116073) [reply requirements/common.txt]: i faced this issue while installing, 
https://github.com/psycopg/psycopg2/issues/594
After upgrading my postgres version and updating my requirements/common.txt  file made it work for me.

File: frontend/dist/views/web/permission-denied.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            <div class="card horizontal ev-md-container ev-card-panel ev-z-depth-5 ">
+                <div class="card-image">
+                    <img src="dist/images/email.png" height="150">
+                </div>
+                <div class="card-stacked">
+                    <div class="card-content">
+                        <p><strong>Unable to create or select a team.</strong><br>
+                            A <strong>verification link</strong> has been sent to your email account.
+                            <br>In order to proceed ahead, please<br>
+                            <button class="button">Confrim Your Email</button>
```
Comment (aka-jain) [inline frontend/dist/views/web/permission-denied.html]: What this button does?

File: frontend/dist/views/web/permission-denied.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    color: white;
+    padding: 15px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 4px 2px;
+    cursor: pointer;
+    border-radius:8px;
+}
```
Comment (aka-jain) [inline frontend/dist/views/web/permission-denied.html]: Hi @16116073, please use existing classes to style the buttons. You can find it in base.scss.

**Comment by 16116073** [comment]
@aka-jain changes done

File: frontend/dist/views/web/permission-denied.html (hunk 3 above)
Comment (16116073) [reply frontend/dist/views/web/permission-denied.html]: done

**Comment by aka-jain** [comment]
@RishabhJain2018 Do we have a feature to resend the mail on clicking resend button?

**Comment by aka-jain** [comment]
@16116073 you have done changes in dist folder but actually you have to do that in src folder. Please revert all your code and do it in src folder files. 
Also, how you have pushed dist files to this PR ? They have listed in gitignore file so they shouldn't be pushed. Can you please check your setup with ReadMe?

**Comment by RishabhJain2018** [comment]
@aka-jain Currently on the master branch branch we don't have it but yes there is a opened pull request for it.

**Comment by RishabhJain2018** [comment]
@aka-jain No, this isn't a GCI task. Also, @Obtin isn't a GCI student. So, we can have more changes here. :-)

**Comment by aka-jain** [comment]
Any update @Obtin

**Comment by Obtin** [comment]
@aka-jain , I have made the changes alreday.
image
image
image

**Pull request closed without merging**